src/components/menu-item/menu-item.tsx

```diff
@@ -172,8 +177,6 @@ export class CalciteMenuItem implements LoadableComponent {
     this.active = this.active || this.editable;
     this.isFocused = this.active;
     // todo just get any nav items in the default slot?
-    // this.hasSubMenu = this.hasSlottedItems();
-
     // for now to detect nesting only working two level for demo.need to just check if it has any parent originating at top lvel
```

Comment:
Can we remove any todo / wip items (I think these are probably mine, ha)

Reply:
i left them to keep a track of whats leftover. Are these irrelevant?

Reply:
removed them.